composer.json

```diff
@@ -68,5 +69,13 @@
 		"exclude": [
 			"log", "cache", "/photo", "/proxy"
 		]
+	},
+	"require-dev": {
```

Comment:
Just to make sure, these requirements fit for all used PHP versions?

Reply:
Yes, I'm using old versions here in order to keep compatibility with PHP 5.6.

Reply:
You can specify multiple version ranges with the `|` (pipe) character. for example, if version `works with PHP 5.6 and version works only with PHP 7, you can specify `^1|^2`, and Composer will pick the right one based on the current version of PHP.

Since it is used for development, we can trust people to run `bin/composer.phar update` themselves.

Reply:
Yes but by default Travis only runs `composer install` so it will use the locked dependencies.
And we don't use any dbunit 3 or 4 feature anyway.

Reply:
Perfect.